src/Components/CreateImageWizard/steps/Oscap/components/OscapProfileInformation.tsx

```diff
@@ -77,29 +83,77 @@ export const OscapProfileInformation = ({
       )}
       {isSuccessOscapProfileInfo && (
         <>
-          <Content>
-            <Content component={ContentVariants.dl} className="review-step-dl">
-              <Content
-                component={ContentVariants.dt}
-                className="pf-v6-u-min-width"
-              >
-                Profile description:
-              </Content>
-              <Content component={ContentVariants.dd}>
-                {oscapProfile?.profile_description}
-              </Content>
-              <Content
-                component={ContentVariants.dt}
-                className="pf-v6-u-min-width"
-              >
-                Reference ID:
-              </Content>
-              <Content
-                data-testid="oscap-profile-info-ref-id"
-                component={ContentVariants.dd}
-              >
-                {oscapProfile?.profile_id}
-              </Content>
+          <Content component={ContentVariants.dl} className="review-step-dl">
+            <Content
+              component={ContentVariants.dt}
+              className="pf-v6-u-min-width"
+            >
+              Profile description
+            </Content>
+            <Content component={ContentVariants.dd}>
+              {oscapProfile?.profile_description}
+            </Content>
+            <Content
+              component={ContentVariants.dt}
+              className="pf-v6-u-min-width"
+            >
+              Reference ID
+            </Content>
+            <Content
+              data-testid="oscap-profile-info-ref-id"
+              component={ContentVariants.dd}
+            >
+              {oscapProfile?.profile_id}
+            </Content>
+            <Content
+              component={ContentVariants.dt}
+              className="pf-v6-u-min-width"
+            >
+              Packages
+            </Content>
+            <Content component={ContentVariants.dd}>
+              {(oscapProfileInfo?.packages ?? []).join(', ')}
```

Comment:
Quick question, do we want to maybe put these in a code block too? Just as a visual cue, since it was kind of hard to spot compared to the others.

![image](https://github.com/user-attachments/assets/ffc0f741-f544-4c1a-8f96-4df0942c2c1a)